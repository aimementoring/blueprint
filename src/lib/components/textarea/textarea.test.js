import React from 'react';
import renderer from 'react-test-renderer';
import Textarea from './textarea';

describe('Textarea', () => {
  it('renders properly', () => {
    const tree = renderer
      .create(
        <Textarea
          placeholder="Write area"
          name="textarea"
          required
          className="myClass"
          onChangeFunction={() => { }}
        />,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
