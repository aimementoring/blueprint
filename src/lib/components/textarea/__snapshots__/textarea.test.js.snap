// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Textarea renders properly 1`] = `
<div
  className=" theme-purple wrapper"
>
  <textarea
    className="false
            myClass input textarea"
    getValidationMessage={[Function]}
    handleValidations={[Function]}
    name="textarea"
    onChange={[Function]}
    onChangeFunction={[Function]}
    placeholder="Write area"
    required={true}
    validationMessage=""
    validations={Array []}
    value=""
  />
</div>
`;
